index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My First Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <h2>welcome</h2>
+        <nav>
+            <i class="fa fa-bars" aria-hidden="true"></i>
+        </nav>
+    </header>
+    <section class="section-1"> 
+        <h1 class="heading-1">Hey there. I’m Josphat</h1>
+            <h2 class="heading-2">I’m a software developer</h2>
+                <p class="par-1"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to
+                </p>
+        <ul class="icons">
+            <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+            <li><i class="fa fa-linkedin" aria-hidden="true"></i></li>
+            <li><i class="fa fa-medium" aria-hidden="true"></i></li>
+            <li><i class="fa fa-github" aria-hidden="true"></i></li>
+            <li><i class="fa fa-angellist" aria-hidden="true"></i></li>
+        </ul>
+    </section>
```

Comment:
- [ ] I would like to suggest that you work on the background images for the section part. One of the project requirements states that it should be the same as that of Figma's template. 

### Your portfolio
![image](https://user-images.githubusercontent.com/77106962/143319793-07d749bc-d6ef-4e4f-810f-b7601c465b80.png)

### Figma's Template
![image](https://user-images.githubusercontent.com/77106962/143320006-89b7285d-27fb-4bb5-b98f-3f76e1f04781.png)

Kindly work on this.